grr/server/grr_response_server/gui/api_plugins/metadata.py

```diff
@@ -51,14 +60,546 @@ class ApiGetOpenApiDescriptionHandler(api_call_handler_base.ApiCallHandler):
   args_type = None
   result_type = ApiGetOpenApiDescriptionResult
 
+  def __init__(self, router):
+    self.router = router
+    self.open_api_obj = None # The main OpenAPI description object.
+    self.schema_objs = None
+
+    self.proto_primitive_types_names = {
```

Comment:
This is not instance-specific, I would move it to a global variable.

Reply:
Moved it, thank you!